src/edgeworkers/ew-handler.ts

```diff
@@ -200,6 +200,27 @@ export async function getContracts() {
   }
 }
 
+export async function getProperties(ewId: string, activeOnly: boolean) {
+  let properties = await cliUtils.spinner(edgeWorkersSvc.getProperties(ewId, activeOnly), `Retrieving properties for EdgeWorker Id ${ewId}...`);
+  if (properties && !properties.isError) {
+    if (properties.properties.length > 0) {
```

Comment:
nitpick. "properties.properties.length" sounds odd. can we say propList.properties.length. Either propList or whatever the api is returning